plugin.cmake

```diff
@@ -0,0 +1,4 @@
+add_web_client_test(
+    flow "${PROJECT_SOURCE_DIR}/plugins/flow/web_client/tests/spec/flowSpec.js"
+    ENABLEDPLUGINS "jobs romanesco flow"
```

Comment:
We should change this from `ENABLEDPLUGINS "jobs romanesco flow"` to just `PLUGIN flow` since flow should bring the others in as dependencies.